modules/index.js

```diff
@@ -163,6 +163,59 @@ export const params = (object) => {
 }
 
 /**
+ * Check agains the final input and options to check if there's missing/misconfigured
+ * properties.
+ */
+export const catchErrors = () => {
+  function checkOptions(input, options) {
+    [
+      {
+        name: 'data',
+        validate: (input, options) => {
+          const verb = (options.method || 'GET').toUpperCase()
+          if (verb === 'GET' || verb === 'HEAD') {
+            return
+          }
+          throw new Error(`Did you try to use 'body: ${JSON.stringify(options.data)}'?`)
+        }
+      },
+      {
+        name: 'body',
+        validate: (input, options) => {
+          const verb = (options.method || 'GET').toUpperCase()
+          if (verb === 'GET' || verb === 'HEAD') {
+            return
+          }
+          function isPojo(obj) {
```

Comment:
Should I use a npm version of this function? It's https://www.npmjs.com/package/is-pojo